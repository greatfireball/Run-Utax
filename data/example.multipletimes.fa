>2160771 Pseudo-nitzschia multiseries
TGCTGGCTGCTGGTTGTTTTGGCTTTGACTGGCACTATTGCCTGTCTCTGCTTAAGTTCTACTGTATAGTACGTGCATAGACGTGGAAGGCTTGACCTGTCTAGTTAAAGACGGCGTTGACACTCTTGTCTATCTCTGGTAGATTGTACTTTTAGTATGAAATATCTGGAGTTTACTATGAGTTTGTTTTCAGCTGTTTTGAAACTGATGGAACGAGCACTTCTATGCCTAACAAACTGTTTACAGTCTGTTATACATTT
>2160771 Pseudo-nitzschia multiseries
TGCTGGCTGCTGGTTGTTTTGGCTTTGACTGGCACTATTGCCTGTCTCTGCTTAAGTTCTACTGTATAGTACGTGCATAGACGTGGAAGGCTTGACCTGTCTAGTTAAAGACGGCGTTGACACTCTTGTCTATCTCTGGTAGATTGTACTTTTAGTATGAAATATCTGGAGTTTACTATGAGTTTGTTTTCAGCTGTTTTGAAACTGATGGAACGAGCACTTCTATGCCTAACAAACTGTTTACAGTCTGTTATACATTT
