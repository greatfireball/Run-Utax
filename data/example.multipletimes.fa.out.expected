2160771	k__super__Eukaryota_2759(100.0),p__Bacillariophyta_2836(100.0),c__Bacillariophyceae_33849(100.0),o__Bacillariales_33851(98.6),f__Bacillariaceae_33852(98.6),g__Pseudo-nitzschia_41953(88.7),s__Pseudo-nitzschia multiseries_37319(85.6)	+
2160771	k__super__Eukaryota_2759(100.0),p__Bacillariophyta_2836(100.0),c__Bacillariophyceae_33849(100.0),o__Bacillariales_33851(98.6),f__Bacillariaceae_33852(98.6),g__Pseudo-nitzschia_41953(88.7),s__Pseudo-nitzschia multiseries_37319(85.6)	+
